bitcoin-death-cross/src/main/java/com/hazelcast/jet/demos/bitcoin/Task1.java

```diff
@@ -0,0 +1,62 @@
+package com.hazelcast.jet.demos.bitcoin;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.Job;
+import com.hazelcast.jet.config.JobConfig;
+import com.hazelcast.jet.demos.bitcoin.job.MovingAverage;
+import com.hazelcast.jet.pipeline.Pipeline;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * <p>This is {@code Task1}. The use of the "{@code @Order}"
+ * annotation tells Spring the order to run these tasks.
+ * </p>
+ * <p>{@code Task1} initiates the Jet job to process Moving
+ * Averages. The Jet job sits waiting for input, which is
+ * produced by a later task ({@code Task4}) simulating a 
+ * real-world feed.
+ * </p>
+ * <p><b>Note:</b> {@code Task1}, {@code Task2} and {@code Task3}
+ * are ordered to run before {@code Task4}. The first three can
+ * be run in any order, all will appear to do nothing until the
+ * fourth starts producing data.
+ * </p>
+ */
+@Component
+@Order(MyConstants.PRIORITY_ONE)
+@Slf4j
+public class Task1 implements CommandLineRunner {
```

Comment:
Can we change the Task1, Task2, Task3, Task4 names to more meaningful ones ?

Reply:
Done. Have renamed `Task1` to `Task1JetJob`, `Task2` to `Task2ChartPanel`, `Task3` to `Task3TopicListener` and `Task4` to `Task4PriceFeed`. I wanted to keep the numbering in there as there is a partial ordering to the tasks, but indicating what each does is a winner.